File: practiceJava/BasicHW/HW4.java

/**
 * Write a description of class HW1 here.
 *
 * @author (your name)
 * @version (a version number or a date)
 */
import java.util.Scanner;
import java.io.*;
public class HW4
{
    public static void main(String args[]){
        int n,x,y,m;
        System.out.println("Please input the number of row: ");
        Scanner rows = new Scanner(System.in);
        n = rows.nextInt();
        System.out.println("Please input the number of column: ");
        Scanner columns = new Scanner(System.in);
        m = columns.nextInt();
        
        for (x=0; x<n; x++){
            for (y=0; y<m; y++){
                System.out.print("*");
                System.out.print("*");
            }
            System.out.println();
        }
        
        
    }
}

File: practiceC/HW4.cpp
#include <stdio.h>

int main() {
  int n,m;
    printf("Please type in the number of rows: ");
    scanf("%d",&n);
    printf("Please type in the number of columns: ");
    scanf("%d",&m);
    for (int x = 0; x < n; x++) {
        for (int y = 0; y < m; y++) {
          printf("*");
          printf("*");
        }
        printf("\n");
      }
}
